### ISSUE TITLE
Add support for MSSQL

### ISSUE THREAD

**Issue by christiaanwesterbeek** [dresende/node-orm2#420]
This looks like the best ORM I can find. For MSSQL there is none useful. I use MSSQL with tedious (and https://npmjs.org/package/mssql that wraps tedious for easier api)

How would I go about to add support for MSSQL to node-orm2? Because this orm already supports so many databases. I imagine I could hook in MSSQL in a similar way as the others have been hooked in.

**Comment by CreativeManix** [comment]
Is MSSQL is supported?

**Comment by dxg** [comment]
Not yet.

**Comment by tcwyjno1** [comment]
@dxg Can you give me some guides.
I want to follow up to make my orm2 to support mssql

**Comment by dxg** [comment]
There is info above about what needs to be done.

**Comment by tcwyjno1** [comment]
@devotis Is your local orm2 support mssql now?

**Comment by devotis** [comment]
I never started to actually use node-orm2 as a dependency. So I don't have anything local that isn't already merged here that adds support for mssql. I use https://github.com/dresende/node-sql-query or http://loopback.io/  in my projects.

**Comment by tcwyjno1** [comment]
Is there any progress with this issue? Hope it can support mssql

**Comment by dxg** [comment]
Nope. I have no interest in adding MSSQL support myself, and until someone opens the required pull requests with tests, MSSQL will not be supported.
